packages/mui-joy/src/Autocomplete/Autocomplete.tsx

```diff
@@ -212,6 +213,7 @@ const AutocompletePopupIndicator = styled(StyledIconButton as unknown as 'button
   slot: 'PopupIndicator',
   overridesResolver: (props, styles) => styles.popupIndicator,
 })<{ ownerState: OwnerState }>(({ ownerState }) => ({
+  alignSelf: 'center',
```

Comment:
StyledIconButton already has `alignItems: center`.

Reply:
`alignItems: center` is for flexbox container but `alignSelf: center` is for flexbox item.

Added `alignSelf: 'center'` here to make the popup indicator stay in the middle of the input.

Reply:
Yes I know that. But I thought there will _always_ be one item inside the popup indicator button.